core/src/main/java/io/grpc/Server.java

```diff
@@ -38,4 +41,57 @@
  * application code or interceptors.
  */
 @ThreadSafe
-public abstract class Server {}
+public abstract class Server {
```

Comment:
I'm not sure it makes sense in this case.  On the client-side, we need the distinction between `Channel` and `ManagedChannel`, but servers *are* managers of channels.  I think keeping the name as `Server` is appropriate here ... WDYT?